test/LibraryService.test.ts

```diff
@@ -0,0 +1,46 @@
+import { initialize, Server } from '@projecttacoma/node-fhir-server-core';
+import { serverConfig } from '../src/config/serverConfig';
+import { cleanUpDb, testSetup } from './utils';
+const supertest = require('supertest');
+
+let server: Server;
+
+const Library: fhir4.Library = {
+  resourceType: 'Library',
+  type: { coding: [{ code: 'logic-library' }] },
+  id: 'library123',
+  status: 'active'
+};
+
+describe('LibraryService', () => {
+  beforeAll(async () => {
+    const config = serverConfig;
+    server = initialize(config);
+    await testSetup([Library]);
+  });
```

Comment:
```suggestion
  });
  
```

As a selfish vim user, it's really hard to navigate tests when there aren't spaces between the different blocks. In general I think it helps the readability overall though